### PR TITLE
Ryan/upstream from c717d6aa7543d4b83395e0552ef2eb311f563aab

### PR DESCRIPTION
**Pull request by r-LaForge**
Linked to earthly branch: https://github.com/earthly/earthly/pull/2285
required fsutil update: https://github.com/earthly/fsutil/pull/9

### CODE REVIEW

**Comment by r-LaForge** [comment]
Attempting to update using earthly branch: https://github.com/earthly/earthly/pull/2285

File: frontend/frontend.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package frontend
 import (
 	"context"
 
+	// earthly-specific this import breaks the mips64 frontend tests due to incorrect types on inode
```
Comment (alexcb) [inline frontend/frontend.go]: nit: can you put this one comment in-line with the import -- it'll make it easier to deal with future conflicts which might re-order the imports

**Comment by alexcb** [comment]
@r-LaForge looks good; can you re-do the merge as a single commit.

When re-resolving conflicts, you can pick all files from `948c5520e02b0d57348bfee5a4db00cb4677e94a` (you're most recent commit).

**Comment by r-LaForge** [comment]
@alexcb absolutely, that's the plan!

Future pr with just the merge commit to follow.

**Comment by r-LaForge** [comment]
This PR is replaced by one with a single merge commit: https://github.com/earthly/buildkit/pull/103

**Pull request closed without merging**